<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUI</class>
 <widget class="QWidget" name="SettingsUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="The title of the settings menu.">Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout_5">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="btnPlaceholderBackups">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string comment="&quot;Backups&quot; section of the Settings.">Backups</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAutoBackups">
     <property name="text">
      <string comment="Whether to do automatic backups when opening a save.">Do Auto Backups:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkAutoBackups"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lvlMaxBackups">
     <property name="text">
      <string comment="The maximum amount of backups the editor will keep.">Maximum Backups</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spnMaxBackups">
     <property name="specialValueText">
      <string comment="An indication when the max backups is unlimited.">Unlimited</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btnPlaceholderItems">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string comment="&quot;Items&quot; section of the Settings.">Items</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblMatrixMode">
     <property name="text">
      <string comment="Whether the editor will render all item icons like a neon sign.">Matrix Mode:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkMatrixMode"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblShowUnobtainable">
     <property name="text">
      <string comment="Show items that are normally unobtainable such as Pure Dracolites or Mega Dash Juice in item search.">Show Unobtainable Items:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkShowUnobtainable"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblSearchAllTabsIncludeItemPouch">
     <property name="text">
      <string comment="Whether to search the player pouch when Search All Tabs is checked.">Include Pouch in Search All Tabs:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="chkSearchAllTabsIncludeItemPouch"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="btnPlaceholderLanguage">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string comment="&quot;Language&quot; section of the Settings.">Language</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblUILanguage">
     <property name="text">
      <string comment="The language the Editor UI will show in.">UI Language:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="cmbUILanguage">
     <property name="placeholderText">
      <string comment="Appears when no UI translations are loaded.">No Translations</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblItemLanguage">
     <property name="text">
      <string comment="The language the items will show in.">Item Language:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="cmbItemLanguage">
     <property name="currentText">
      <string notr="true">Use game language</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string comment="Whether the items should be displayed in the game's language.">Use game language</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">日本語</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Français</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Italiano</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Deutsch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Español</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Português do Brasil</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Polski</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Русский</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">한국어</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">繁體中文</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">简体中文</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Arabic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QPushButton" name="btnPlaceholderQualityOfLife">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string comment="&quot;Quality of Life&quot; section of the Settings.">Quality of Life</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblDarkMode">
     <property name="text">
      <string comment="Dark mode for the editor UI.">Dark Mode:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="chkDarkMode"/>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QLabel" name="lblRestartDisclaimer">
     <property name="text">
      <string comment="Indicate options which need a restart.">* Will apply on restart.</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>chkAutoBackups</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetAutoBackups(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnMaxBackups</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetMaxBackups(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMatrixMode</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetMatrixMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDarkMode</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetDarkMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShowUnobtainable</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetShowUnobtainableItems(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchAllTabsIncludeItemPouch</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetIncludePouchInSearch(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbUILanguage</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetUILanguage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbItemLanguage</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsUI</receiver>
   <slot>SetItemLanguage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetAutoBackups(int)</slot>
  <slot>SetMaxBackups(int)</slot>
  <slot>SetMatrixMode(int)</slot>
  <slot>SetShowUnobtainableItems(int)</slot>
  <slot>SetIncludePouchInSearch(int)</slot>
  <slot>SetDarkMode(int)</slot>
  <slot>SetItemLanguage(int)</slot>
  <slot>SetUILanguage(int)</slot>
 </slots>
</ui>
